webapp-django/crashstats/crashstats/models.py

```diff
@@ -652,53 +652,53 @@ class ProcessedCrash(SocorroMiddleware):
     }
 
     API_WHITELIST = (
-        'ReleaseChannel',
+        'additional_minidumps',
+        'addons',
         'addons_checked',
         'address',
+        'app_notes',
         'build',
         'client_crash_date',
         'completeddatetime',
+        'cpu_info',
         'cpu_name',
+        'crashedThread',
+        'crash_time',
         'date_processed',
+        'distributor',
         'distributor_version',
         'dump',
         'flash_version',
         'hangid',
+        'hang_type',
         'id',
+        'install_age',
+        'java_stack_trace',
+        'json_dump',
         'last_crash',
+        'mdsw_status_string',
         'os_name',
         'os_version',
+        'pluginFilename',
+        'pluginName',
+        'pluginVersion',
+        'processor_notes',
         'process_type',
         'product',
+        'productid',
         'reason',
         'release_channel',
+        'ReleaseChannel',
         'signature',
+        'startedDateTime',
         'success',
+        'topmost_filenames',
         'truncated',
+        'upload_file_minidump_*',
         'uptime',
         'uuid',
         'version',
-        'install_age',
-        'startedDateTime',
-        'java_stack_trace',
-        'crashedThread',
-        'cpu_info',
-        'pluginVersion',
-        'hang_type',
-        'distributor',
-        'topmost_filenames',
-        'additional_minidumps',
-        'processor_notes',
-        'app_notes',
-        'crash_time',
         'Winsock_LSP',
-        'productid',
-        'pluginFilename',
-        'pluginName',
-        'addons',
-        'json_dump',
-        'upload_file_minidump_*',
-        'mdsw_status_string',
```

Comment:
I annoyingly alphabetized these. It was impossible to see what was there and what wasn't.